console/console-init/ui/src/modules/device-detail/components/DeviceInfoGateways/DeviceInfoGateways.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import {
+  Grid,
+  GridItem,
+  Card,
+  CardBody,
+  CardHeader,
+  Title,
+  Button,
+  Text,
+  TextVariants
+} from "@patternfly/react-core";
+
+interface IDeviceInfo {
+  id: string;
+  name: string;
+}
+
+export interface IDeviceInfoGatewaysProps {
+  deviceList?: IDeviceInfo[];
+}
+
+export const DeviceInfoGateways: React.FC<IDeviceInfoGatewaysProps> = ({
+  deviceList
+}) => {
+  const DeviceNotFound = () => (
+    <Text component={TextVariants.p}>
+      There is no gateways for this device. This device is connected to the
```

Comment:
Just a typo - There *are no gateways....

Reply:
In wireframe its `there is no`. `There are no` is plural and `there is no` is singular. I think it is fine as singular.